Carthage/Checkouts/SwiftCheck

```diff
@@ -1 +1 @@
-Subproject commit f2141461f015315cc583222f0bb30ae07da8aad8
+Subproject commit 983b5954f221d36efe3469e8a5155d953034994a
```

Comment:
Please don't manually update the submodules.  Edit the cartfile and run `carthage update --use-submodules` then commit the result.